integration/airflow/openlineage/airflow/extractors/manager.py

```diff
@@ -0,0 +1,56 @@
+import logging
+from typing import Optional
+
+from openlineage.airflow.extractors import TaskMetadata, BaseExtractor, Extractors
+from openlineage.airflow.utils import get_job_name
+
+
+class ExtractorManager:
+    """Class abstracting management of custom extractors."""
+    def __init__(self):
+        self.extractors = {}
+        self.extractor_mapper = Extractors()
+        self.log = logging.getLogger()
+
+    def extract_metadata(self, dagrun, task, complete: bool, task_instance=None) -> TaskMetadata:
+        extractor = self._get_extractor(task)
+        task_info = f'task_type={task.__class__.__name__} ' \
+            f'airflow_dag_id={task.dag_id} ' \
+            f'task_id={task.task_id} ' \
+            f'airflow_run_id={dagrun.run_id} '
+        if extractor:
```

Comment:
_Minor_: I would briefly document the default behavior if an extractor is not defined for a task / operator.